# center: x,y,z  size: x,y,z  omega:yaw
box:                  [ 4.743 ,  1.730,  0.3 , 0.6, 0.6, 0.6,       2.2698,
                        6.631 ,  1.125,  0.3 , 0.6, 0.6, 0.6,       2.2461,
                        7.044 , -1.911,  0.3 , 0.6, 0.6, 0.6,       1.5614,
                        5.848 , -1.874,  0.37, 0.22,0.35,0.74,      0,
                        6.063 , -1.874,  0.56, 0.21,0.35,0.37,      0,
                        6.243 , -1.874,  0.53, 0.15,0.35,0.43,      0,
                        6.423 , -1.874,  0.56, 0.21,0.35,0.37,      0,
                        6.638 , -1.874,  0.37, 0.22,0.35,0.74,      0,

                        5.446 , -1.880,  0.3 , 0.6, 0.6, 0.6,       1.5739,
                      ]

#  6.243 , -1.874,  0.3, 0.22,0.35,0.74,    0,

# map_size  xmin xmax ymin ymax zmin zmax
map_size:    [-5, 10,
                       -5, 10,
                       0, 2]